### PR TITLE
Scottx611x/fix svg 404s

### LINKED ISSUE #1967: File not found error for /static/images/icons.svg
* Specific code commit: release v1.5.8
* Version of the web browser and OS: any
* Environment where the error occurred (Vagrant VM and site conf mode or AWS instance): AWS

### Steps to reproduce
Load the launchpad in a web browser

### Observed behavior
Error message in the browser console:
```
Failed to load resource: the server responded with a status of 404 (NOT FOUND) /static/images/icons.svg#cursor-default
```

### Expected behavior
No errors in the console.

### PR DESCRIPTION
**Pull request by scottx611x**
Fixes #1967 

Used [this link](https://stackoverflow.com/a/25609356/6031066) for guidance. Not sure if there is a better way.

### CODE REVIEW

File: refinery/ui/source/js/commons/treemap/template.html [hunk 1]
```diff
@@ -50,7 +50,8 @@
         <svg class="icon-svg">
           <use
             xmlns:xlink="http://www.w3.org/1999/xlink"
-            xlink:href="/static/images/icons.svg#maximize"></use>
+            xlink:href=""
```
Comment (hackdna) [inline refinery/ui/source/js/commons/treemap/template.html]: Why is this needed?

Comment (scottx611x) [reply refinery/ui/source/js/commons/treemap/template.html]: Not sure, but it it doesn't work without it. 
I followed the advice from [here](https://stackoverflow.com/a/25609356/6031066).
